report-job-status/action.yml

```diff
@@ -74,18 +99,37 @@ runs:
                 JSON.stringify(steps[key])
             }
           })
+          
+          // Try to find the job ID by its name
+          // In matrix workflows it will work only if exact name was provided
+          // in `job-name` input.
+          for (job of jobContext.jobs) {
+            if (job.name == failedJobName ) {
+              jobId = job.id
+            }
+          } 
+          if (jobId != '') {
```

Comment:
Looks like we use double quotes everywhere in JS.

Reply:
fixed